packages/eslint-config-airbnb-base/whitespace.js

```diff
@@ -0,0 +1,73 @@
+const baseConfig = require('.');
+const entries = require('object.entries');
+const { CLIEngine } = require('eslint');
+
+function onlyErrorOnRules(rulesToError, config) {
+  const errorsOnly = { ...config };
```

Comment:
This object spread property in nodejs is introduced at version 8+. If this file is meant to be run with nodejs 8+, then we don't need to depend on more lib e.g. `object.entries`, `Object.entries` already exists there.

I can make a PR to clean this. What do you think, @ljharb?

Reply:
Even if we convert this file to be compatible with nodejs 4 which is declared in package.json, we still don't need this extra dependency, since eslint already depends on lodash, then we can add it too and use _.forEach.

Reply:
Whoops, this should use `object.assign` instead of spread.

There’s no cost to an extra dev dep, and I’d prefer not to use lodash.

Reply:
I see, then we need to change array destructuring at line 10 too.

I'll make a PR, ok?

Reply:
Sure.